packages/client/package.json

```diff
@@ -31,7 +31,7 @@
     "build:ts": "tsc --build",
     "build:cjs": "rollup --config rollup.config.js",
     "coverage": "nyc report --reporter=text-lcov > coverage.lcov && npx codecov",
-    "typedoc": "typedoc --entryPoints src  --out ../docs/client",
+    "typedoc": "typedoc --entryPoints src  --out ../../docs/client",
```

Comment:
unrelated fix?

Reply:
Yes, a hangover from when we moved everything into the `packages` directory. These should be written to the root.